Modules/Core/Common/wrapping/test/itkPointSetTest.py

```diff
@@ -0,0 +1,52 @@
+# ==========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          https://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ==========================================================================*/
+
+import itk
+import unittest
+import numpy as np
+
+class PointSetTestCase(unittest.TestCase):
+    """Tests itk.PointSet"""
+
+    def test_SetPointsByCoordinates_from_NumPy_array(self) -> None:
+        """Tests the SetPointsByCoordinates method, having a NumPy array as input"""
+
+        # The pixel type is irrelevant for this test, just use unsigned char.
+        PixelType = itk.UC
```

Comment:
> Instead of Unsigned Char, can we instead use float ?  Because that will be the most used datatype.

@PranjalSahu Thanks for the suggestion, but isn't UC a common pixel type as well? We could extend the test to try any possible pixel type, but it's irrelevant to the functionality of `SetPointsByCoordinates` anyway. `SetPointsByCoordinates` is just about the points of a PointSet. It's not about the pixels.

Note that this is _not_ about the coordinate type. The coordinate type is defined by the "MeshTraits" of a PointSet type. (By default, it is 32-bit float, independent of the pixel type.)

Reply:
For the record, by default, `itk.PointSet` supports double, float, short, unsigned char, unsigned int, itk::Array, itk::Matrix, itk::Point, and itk::Vector as pixel type, according to `itk.PointSet.GetTypes()`:

```python
>>> print(itk.PointSet.GetTypes())
<itkTemplate itk::PointSet>
Options:
  [<itkCType double>, 2]
  [<itkCType double>, 3]
  [<itkCType float>, 2]
  [<itkCType float>, 3]
  [<itkCType signed short>, 2]
  [<itkCType signed short>, 3]
  [<itkCType unsigned char>, 2]
  [<itkCType unsigned char>, 3]
  [<itkCType unsigned int>, 2, <class 'itk.itkDefaultStaticMeshTraitsPython.itkDefaultStaticMeshTraitsUI22FFUI'>]
  [<itkCType unsigned int>, 3, <class 'itk.itkDefaultStaticMeshTraitsPython.itkDefaultStaticMeshTraitsUI33FFUI'>]
  [<class 'itk.itkArrayPython.itkArrayD'>, 2]
  [<class 'itk.itkArrayPython.itkArrayD'>, 3]
  [<class 'itk.itkMatrixPython.itkMatrixD22'>, 2]
  [<class 'itk.itkMatrixPython.itkMatrixD33'>, 3]
  [<class 'itk.itkPointPython.itkPointD2'>, 2, <class 'itk.itkDefaultStaticMeshTraitsPython.itkDefaultStaticMeshTraitsD22DDD'>]
  [<class 'itk.itkPointPython.itkPointD3'>, 3, <class 'itk.itkDefaultStaticMeshTraitsPython.itkDefaultStaticMeshTraitsD33DDD'>]
  [<class 'itk.itkVectorPython.itkVectorF3'>, 3]
None

```